src/gui/screen_home.cpp

```diff
@@ -62,7 +62,7 @@ struct screen_home_data_t {
 #define pw ((screen_home_data_t *)screen->pdata)
 
 static bool find_latest_gcode(char *fpath, int fpath_len, char *fname, int fname_len);
-void screen_home_disable_print_button(screen_t *screen);
+void screen_home_disable_print_button(screen_t *screen, int disable);
```

Comment:
prefer bool in c++

Reply:
I know it is preferable use boolean in C++, but here I used int, because I can set a flags (f_enabled, f_disabled) right away and not after i recast it or using ternary operator to set 1 || 0 in those flags.